DOMEvents/event-types/events-keyboardevents/KeyboardEvent.modifiers-manual.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>KeyboardEvent.getModifierState() and 'AltGraph' modifier key</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+async_test(function() {
+  var event = "keydown";
+
+  window.onload = this.step_func(function() {
+    var target = document.getElementById("target");
+    target.addEventListener(event, this.step_func(function test_event(e) {
+      assert_equals(e.type, event);
+      if (e.getModifierState("AltGraph")) {
+        target.removeEventListener(event, test_event, true);
+        this.done();
+      }
+    }), true);
+  });
+});
+</script>
+
+<p>Some operating systems simulate the 'AltGraph' modifier key with the
+combination of the 'Alt' and 'Control' modifier keys. Implementations are
+encouraged to use the 'AltGraph' modifier key.</p>
+
+<p><input id="target" value=""></p>
+<ol class=instructions>
+  <li>Click in the above textbox using mouse
+  <li>Press and hold down '{CTRL}' key and then press '{ALT}' key on the keyboard
```

Comment:
Firefox also fails. And I don't think that it makes sense to treat Ctrl and Alt are inactive in such case because except Europe keyboard layouts, AltGr isn't so major. For example, Asian web developers may want to use Ctrl+Alt+foo for a shortcut key because most users use their country's primary keyboard layout or US keyboard layout.

So, from a point of view of compatibility with existing browsers, Ctrl + Alt on Windows should activate all of Alt, Control and AltGraph.